core/app/models/spree/stock/package.rb

```diff
@@ -4,14 +4,14 @@ class Package
       attr_reader :stock_location, :contents
       attr_accessor :shipping_rates
 
-      def initialize(stock_location, contents=[])
+      def initialize(stock_location, contents = nil)
```

Comment:
`contents = Set.new` here would cause things to fail if nil was passed in, as it wouldn't be overriden to be a `Set`.